### ISSUE TITLE
Focal Loss for Dense Object Detection

### ISSUE THREAD

**Issue by joyhuang9473** [number9473/nn-algorithm#156]
# Focal Loss for Dense Object Detection #

- Author: Tsung-Yi Lin, Priya Goyal, Ross Girshick, Kaiming He, Piotr Dollár
- Origin: https://arxiv.org/abs/1708.02002
- Related:
  - https://github.com/yhenon/pytorch-retinanet
  - https://github.com/c0nn3r/RetinaNet
  - Implementation for softmax, mxnet: https://github.com/unsky/focal-loss
  - Implementation for sigmoid form: https://github.com/sciencefans/Focal-Loss
  - Implementation for softmax form: https://github.com/zimenglan-sysu-512/Focal-Loss

> RetinaNet

**Comment by joyhuang9473** [comment]
@weiliu89 comment about focal loss

https://github.com/weiliu89/caffe/issues/720#issuecomment-324198845

**Comment by joyhuang9473** [comment]
@chuanqi305 's version of focal loss for SSD https://github.com/chuanqi305/FocalLoss

https://github.com/chuanqi305/MobileNet-SSD/issues/15#issuecomment-326747858
